templates/pool.conf.j2

```diff
@@ -7,7 +7,7 @@
 ; The address on which to accept FastCGI requests.
 ; Valid syntaxes are: 'ip.add.re.ss:port', 'port', '/path/to/unix/socket'.
 ; This option is mandatory for each pool.
-listen = {{ php_fpm_pool.listen | mandatory }}
```

Comment:
Why do you want to remove the mandatory vars

Reply:
Because its implicit and error messages don't show which mandatory variable is undefined